doc/architecture/decisions/0009-use-pundit-to-manage-permissions.md

```diff
@@ -0,0 +1,22 @@
+# 9. Use Pundit to manage permissions
+
+Date: 2019-11-22
+
+## Status
+
+Accepted
+
+## Context
```

Comment:
Is it worth documenting the option of using Auth0 of the master for roles and then why we don't think it is preferable?